### DISCUSSION TITLE
Metrics values are not show decimals

### DISCUSSION THREAD

**Discussion by ranierif** [laravel/nova-issues#4261] [Q&A] [answered]
- Laravel Version: 9.2
- Nova Version: 4.5.4
- PHP Version: 8.1
- Database Driver & Version: MYSQL 8.0
- Operating System and Version: Mac OSX
- Browser type and version: Chrome 100.0.4896.75

### Description: 

After trying many ways to format the Metrics values, I can't see the decimals of any value. If I have a value of 1.1, only 1 is shown.

### Detailed steps to reproduce the issue on a fresh Nova installation:

- The function of TREND:

```php
<?php

namespace App\Nova\Metrics;

use App\Models\Deposit;
use Laravel\Nova\Http\Requests\NovaRequest;
use Laravel\Nova\Metrics\Trend;

class DepositPerDay extends Trend
{
    /**
     * Calculate the value of the metric.
     *
     * @param  \Laravel\Nova\Http\Requests\NovaRequest  $request
     * @return mixed
     */
    public function calculate(NovaRequest $request)
    {
        return $this->sumByDays(
            $request,
            Deposit::class,
            'amount'
        )->format('0.000')->showSumValue();
    }
}
```

- And this is how it shows in the View:

<img width="510" alt="Screen Shot 2022-05-16 at 21 47 12" src="https://user-images.githubusercontent.com/11886513/168670837-1b24572d-338a-4b72-9b5d-b5ffe50d4401.png">

**Comment by crynobone** [comment, marked as answer]
You can add `precision($precision = 0, $mode = PHP_ROUND_HALF_UP)` to configure how rounding work with the results.

 ```php
return $this->precision(2)
            ->sumByDays(
                $request,
                Deposit::class,
                'amount'
            )
            ->format('0.000')
            ->showSumValue();
```

**Reply by ranierif** [reply]
Thanks for your reply @crynobone! This solution resolve my case.

I think is a good option add this on the documentation of metrics.
